src/client/pages/Varen/VarenDetailExploitatie.test.tsx

```diff
@@ -363,5 +363,73 @@ describe('<VarenDetail />', () => {
         'true'
       );
     });
+
+    it('Disables action buttons on Varen vergunning exploitatie when vergunning has EV change request in progress', () => {
+      const screen = render(
+        <Component
+          state={getTestState([
+            exploitatieDecision,
+            {
+              ...exploitatieDecision,
+              processed: false,
+              decision: 'Nog niet bekend',
+              caseType: 'Varen vergunning exploitatie Wijziging vaartuignaam',
+              vesselNameNew: 'test',
+            },
+          ])}
+        />
+      );
+      const btnWijzigen = screen.queryByRole('link', {
+        name: BTN_TEXT_WIJZIGEN_EXPLOITATIE,
+      });
+      expect(btnWijzigen).not.toBeInTheDocument();
+
+      const btnWijzigenDisabled = screen.getByText(
+        BTN_TEXT_WIJZIGEN_EXPLOITATIE_HAS_WIJZIGING
+      );
+      expect(btnWijzigenDisabled).not.toHaveAttribute('href');
+      expect(btnWijzigenDisabled).toHaveAttribute('aria-disabled', 'true');
+
+      const btnLigplaatsAanvragen = screen.queryByRole('link', {
+        name: BTN_TEXT_LIGPLAATS_AANVRAGEN,
+      });
+      expect(btnLigplaatsAanvragen).not.toBeInTheDocument();
+
+      const btnLigplaatsAanvragenDisabled = screen.getByText(
+        BTN_TEXT_LIGPLAATS_AANVRAGEN
+      );
+      expect(btnLigplaatsAanvragenDisabled).not.toHaveAttribute('href');
+      expect(btnLigplaatsAanvragenDisabled).toHaveAttribute(
+        'aria-disabled',
+        'true'
+      );
+    });
+
+    it('Does not modify the button text if another vergunning has an EV change request in progress', () => {
+      const screen = render(
+        <Component
+          state={getTestState([
+            exploitatieDecision,
+            {
+              ...exploitatieDecision,
+              vergunningKenmerk: `${exploitatieDecision.vergunningKenmerk}-notEqual`,
+              processed: false,
+              decision: 'Nog niet bekend',
```

Comment:
Ga je hier nog wat meer doen? Zo niet zou ik liever zien dat er geen decision is, `null`